cookielaw/templates/cookielaw/banner.html

```diff
@@ -1,12 +1,13 @@
 {% load i18n %}
 
-<div id="CookielawBanner" {% if js_mode %}class="BannerJsMode" style="display:none"{% endif %}>
+<div id="CookielawBanner" class="{% if js_mode %}BannerJsMode style='display:none'{% endif %} fixedbottom">
```

Comment:
Hi @ipenev92,

There is a little error here: the quotation mark of the `class` attribute is not well closed.